src/main.py

```diff
@@ -276,9 +288,13 @@ def main():
         except Exception as e:
             logging.error(e)
             time.sleep(10)
+        if killhandler.killed==True:
```

Comment:
This is a nitpick but please change to just `if killhandler.killed:`

That's the slightly more standard Python way of checking a Boolean value